src/components/Picker/index.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import MenuItem from '@material-ui/core/MenuItem';
+import FormControl from '@material-ui/core/FormControl';
+import Select from '@material-ui/core/Select';
+
+const styles = theme => ({
+  root: {
+    display: 'flex',
+    flexWrap: 'wrap',
+  },
+  formControl: {
+    margin: theme.spacing.unit,
+    minWidth: 120,
+  },
+  selectEmpty: {
+    marginTop: theme.spacing.unit * 2,
+  },
+});
+
+const Picker = ({
+  classes, identifier, onSelection, options, selectedValue,
+}) => (
+  <form className={classes.root} autoComplete="off">
+    <FormControl className={classes.formControl}>
+      <Select
+        value={selectedValue}
+        onChange={onSelection}
+        inputProps={{ name: identifier, id: identifier }}
+      >
+        <MenuItem value={selectedValue}><em>{selectedValue}</em></MenuItem>
+        {options.map(option => (
+          <MenuItem key={option} value={option}>{option}</MenuItem>
+        ))}
+      </Select>
+    </FormControl>
```

Comment:
You can use the `TextField` component `<TextField select ... />` instead of `FormControl` & `Select`. The `TextField` component is essentially a wrapper around those components. https://material-ui.com/api/text-field/#advanced-configuration. Because of that, I don't think you need to create a `Picker` component.